Shadowrun6th-German/translations/fr.json

```diff
@@ -1,364 +1,364 @@
-	"health_air": "Heilung (Luft)",
-	"health_beast": "Heilung (Tier)",
-	"health_blood": "Heilung (Blut)",
-	"health_earth": "Heilung (Erde)",
-	"health_fire": "Heilung (Feuer)",
-	"health_guardian": "Heilung (Beschützer)",
-	"health_man": "Heilung (Mensch)",
-	"health_plant": "Heilung (Pflanze)",
-	"health_task": "Heilung (Helfer)",
-	"health_water": "Heilung (Wasser)",
-	"heavypistol": "Schwere Pistole",
+	"gear": "Equipements",
+	"guardian": "Gardien",
+	"health_air": "Santé (Air)",
+	"health_beast": "Santé (Bête)",
+	"health_blood": "Santé (Sang)",
+	"health_earth": "Santé (Terre)",
+	"health_fire": "Santé (Feu)",
+	"health_guardian": "Santé (Gardien)",
+	"health_man": "Santé (Homme)",
+	"health_plant": "Santé (Plante)",
+	"health_task": "Santé (Tâche)",
+	"health_water": "Heilung (Eau)",
+	"heavypistol": "Pistolet Lourd",
 	"hm": "HM",
 	"hmam": "HM/AM",
 	"hmsm": "HM/SM",
 	"hmsmam": "HM/SM/AM&nbsp;&nbsp;",
-	"holdout": "Holdout-Pistolen",
-	"hide": "Verbergen?",
-	"illusion_air": "Illusion (Luft)",
-	"illusion_beast": "Illusion (Tier)",
-	"illusion_blood": "Illusion (Blut)",
-	"illusion_earth": "Illusion (Erde)",
-	"illusion_fire": "Illusion (Feuer)",
-	"illusion_guardian": "Illusion (Beschützer)",
-	"illusion_man": "Illusion (Mensch)",
-	"illusion_plant": "Illusion (Pflanze)",
-	"illusion_task": "Illusion (Helfer)",
-	"illusion_water": "Illusion (Wasser)",
-	"ini": "Verteidigungspool",
-	"iniastral": "Initiative (Astralraum)",
-	"inidice": "Initiativewürfel",
-	"iniphys": "Initiative (physisch)",
-	"inimatrix": "Initiative (Heißes Sim)",
+	"holdout": "Holdout-Pistol",
+	"hide": "Caché?",
+	"illusion_air": "Illusion (Air)",
+	"illusion_beast": "Illusion (Bête)",
+	"illusion_blood": "Illusion (Sang)",
+	"illusion_earth": "Illusion (Terre)",
+	"illusion_fire": "Illusion (Feu)",
+	"illusion_guardian": "Illusion (Gardien)",
+	"illusion_man": "Illusion (Homme)",
+	"illusion_plant": "Illusion (Plante)",
+	"illusion_task": "Illusion (Tâche)",
+	"illusion_water": "Illusion (Eau)",
+	"ini": "Initiative",
+	"iniastral": "Initiative (Astral",
+	"inidice": "Dé d'Initiative",
+	"iniphys": "Initiative (physique)",
+	"inimatrix": "Initiative (Hot Sim)",
 	"initiative": "Initiative",
 	"intuition": "Intuition",
 	"intuition_short": "INT",
-	"int_wil": "INT + WIL",
-	"judgeintent": "Menschenkenntnis",
-	"lightpistol": "Leichte Pistole",
-	"liftcarry": "Heben & Tragen",
-	"logic": "Logik",
+	"int_wil": "INT + VOL",
+	"judgeintent": "Juger les intentions",
+	"lightpistol": "Pistolet léger",
+	"liftcarry": "Porter & Soulever",
+	"logic": "Logic",
 	"logic_short": "LOG",
 	"log_int": "LOG + INT",
-	"machinegun": "Maschinengewehr",
-	"machinepistol": "Automatikpistole",
+	"machinegun": "Mitrailette",
+	"machinepistol": "Pistolet Automatique",
 	"magic": "Magie",
 	"magic_short": "MAG",
-	"man": "Mensch",
-	"manipulation_air": "Manip. (Luft)",
+	"man": "Homme",
+	"manipulation_air": "Manip. (Air)",
 	"manipulation_beast": "Manip. (Tier)",
```

Comment:
Not sure if Beast and Blood are used at the current state of 6e but translating that should make it future proof.